addons/cert-manager/metadata.yaml

```diff
@@ -14,3 +14,6 @@ deployTo:
 dependencies:
   - name: fluxcd
 
+system:
+  vela: ">=1.5.0-beta.3"
```

Comment:
Please change the addon version as well (e.g 1.7.1 -> 2.0.0). Requiring newer vela versions is a breaking change. We want user that are using pre-1.5.0 versions to be able use older versions of cert-manager by making 1.7.1 available instead overwriting it.

Reply:
yes, provide a new version for cert-manager addon